spec/controllers/account/profiles_controller_spec.rb

```diff
@@ -25,6 +25,13 @@
       expect(user.first_name).to eq(user_valid_params[:first_name])
     end
 
+    it "should update the user's avatar" do
+      patch :update, params: { id: user.id, user: user_valid_params }
+      expect(user.avatar.attached?).to eq(false)
```

Comment:
[rspec `be_` matcher!](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/be-matchers)

Reply:
it's minor. Personally I prefer to see actual value over readable text